### PR TITLE
remove types from linear operators

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

**Comment by dpo** [comment]
Updated.

**Comment by abelsiqueira** [comment]
:robot: Testing breakage of this commit

| Package Name | master | Tagged |
|--|--|--|
| AmplNLReader |  |  |
| CaNNOLeS |  |  |
| CUTEst |  |  |
| QuadraticModels |  |  |
| NLPModelsIpopt |  |  |
| NLPModelsJuMP |  |  |
| SolverTools |  |  |

File: Project.toml [hunk 1]
```diff
@@ -14,5 +14,5 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 [compat]
 FastClosures = "0.2.1, 0.3.0"
 ForwardDiff = "0.9.0, 0.10.0"
-LinearOperators = "0.5.0, 0.6.0"
+LinearOperators = "0.7.0, 2"
```
Comment (abelsiqueira) [inline Project.toml]: The syntax is just "0.7.0" now.
"0.7.0, 2" means "0.7.x" and "2.x".

Comment (abelsiqueira) [reply Project.toml]: Actually "0.7.0, 1.0" to support 1.0 as well.

Comment (dpo) [reply Project.toml]: If I understand [the docs](https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1) correctly, "0.7.0, 2" means "from 0.7.0 up to, and not including, 2.0.0".

Comment (dpo) [reply Project.toml]: I guess it should be "0.7.0, 1".

**Comment by abelsiqueira** [comment]
:robot: Testing breakage of this commit

| Package Name | master | Tagged |
|--|--|--|
| AmplNLReader |  |  |
| CaNNOLeS |  |  |
| CUTEst |  |  |
| QuadraticModels |  |  |
| NLPModelsIpopt |  |  |
| NLPModelsJuMP |  |  |
| SolverTools |  |  |

**Comment by abelsiqueira** [comment]
Thank you.

**Pull request merged**